HCSStarRatingView/HCSStarRatingView.h

```diff
@@ -42,5 +44,13 @@ IB_DESIGNABLE
 @property (nonatomic, strong) IBInspectable UIImage *emptyStarImage;
 @property (nonatomic, strong) IBInspectable UIImage *halfStarImage;
 @property (nonatomic, strong) IBInspectable UIImage *filledStarImage;
+
+@property (nonatomic, weak) IBOutlet id<HCSStarRatingViewDelegate> delegate;
 @end
 
+@protocol HCSStarRatingViewDelegate <NSObject>
+
+@optional
+- (void)userFinishedRatingOnView:(HCSStarRatingView*)ratingView withRating:(CGFloat)value;
```

Comment:
Cool, in that case I'll merge the PR and update the pod once this is changed. Thanks! 😄